django_social_share/templatetags/social_share.py

```diff
@@ -111,3 +112,21 @@ def post_to_gplus(context, obj_or_url=None, link_text='Post to Google+'):
     context = post_to_gplus_url(context, obj_or_url)
     context['link_text'] = link_text
     return context
+
+
+@register.simple_tag(takes_context=True)
+def post_to_mail_url(context, text, subject, obj_or_url=None):
```

Comment:
I would switch the order of the arguments to `context, subject, text, obj_or_url=None`, so that you can't have email without a subject. Same for `post_to_mail`